### PR TITLE
chore(annotations): remove span_evaluations resolvers

### LINKED ISSUE #4180: cleanup and remove evaluation resolvers

### PR DESCRIPTION
**Pull request by Parker-Stafford**
resolves #4180 

Frontend is fully migrated to spanAnnotations so this removes spanEvaluations from the graphql portion of the server

documentEvaluations still remain -

### CODE REVIEW

File: tests/server/api/routers/v1/test_annotations.py [hunk 1]
```diff
@@ -308,7 +308,7 @@ def assert_summaries(
     ) -> Callable[[bool, float], Awaitable[None]]:
         async def _(exist: bool, score_offset: float = 0) -> None:
             expected = mean_score + score_offset
-            for summaries, names in ((anno_summaries, anno_names), (eval_summaries, eval_names)):
+            for summaries, names in ((anno_summaries, anno_names), ([], eval_names)):
```
Comment (RogerHYang) [inline tests/server/api/routers/v1/test_annotations.py:311]: you can just remove the `eval_names` altogether
```suggestion
            for summaries, names in ((anno_summaries, anno_names),):
```

**Review by mikeldking — APPROVED** [review]
🧼

File: tests/server/api/routers/v1/test_annotations.py (hunk 1 above)
Comment (RogerHYang) [inline tests/server/api/routers/v1/test_annotations.py:311]: i think you want this instead?
```suggestion
           summaries, names = anno_summaries, anno_names + eval_names
```

**Review by RogerHYang — APPROVED** [review]
did you also want to remove `evaluation_summaries` and its cache?

**Comment by Parker-Stafford** [comment]
> did you also want to remove `evaluation_summaries` and its cache?

oh i think we can yeah let me see

**Pull request merged**